src/Components/People.js

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+
+export class People extends Component {
+    constructor(){
+        super()
+
+        this.state = {
+            input: "",
+            people: [],
+            name: "",
+            age: "",
+            gender: "",
+            isError: false,
+        }
+    }
+
+    componentDidMount () {
+        this.getPeople()
```

Comment:
I love your strategy to fetch a list of people once when the component mounts and then search this pre-loaded list any time someone submits a new name!